lldb/source/Core/FormatEntity.cpp

```diff
@@ -1245,9 +1245,10 @@ bool FormatEntity::Format(const Entry &entry, Stream &s,
             llvm::Triple::OSType ostype = arch.IsValid()
                                               ? arch.GetTriple().getOS()
                                               : llvm::Triple::UnknownOS;
-            if ((ostype == llvm::Triple::FreeBSD) ||
-                (ostype == llvm::Triple::Linux) ||
-                (ostype == llvm::Triple::NetBSD)) {
+            if (ostype == llvm::Triple::Linux ||
```

Comment:
OSes were in alpha order before -- it doesn't matter too much but both consistency across the code base and avoiding churn are good goals. Is the order is generally Linux then BSDs in alpha order across the tree?

Reply:
There are some cases like that in some areas. I try to keep the BSD's grouped together and in order of DragonFly, FreeBSD, NetBSD, OpenBSD where relevant. I took out that part.